### PR TITLE
Fix safeAreaInsets offset on BLIK component

### PR DESCRIPTION
**Pull request by nauaros**
## Summary
Fix bug on BLIK component where bottom offset is missing.

### CODE REVIEW

**Comment by mohammedDehairy** [comment]
unit tests are failing, can you fix it?

File: AdyenDropIn/Presentation/WrapperViewController.swift [hunk 1]
```diff
@@ -100,6 +112,7 @@ extension ModalViewController {
         guard preferredContentSize != .zero else { return frame }
 
         let bottomPadding = max(abs(keyboardRect.height), view.safeAreaInsets.bottom)
+        print("Bottom padding: \(view.safeAreaInsets)")
```
Comment (mohammedDehairy) [inline AdyenDropIn/Presentation/WrapperViewController.swift:115]: please remove this

File: AdyenDropIn/Presentation/WrapperViewController.swift [hunk 2]
```diff
@@ -52,36 +48,51 @@ internal final class WrapperViewController: UIViewController {
...
-        view.layoutIfNeeded()
-    }
 
-    fileprivate func positionContent(_ child: ModalViewController) {
+    // MARK: - Private
+
+    private func setupChildViewController() {
         addChild(child)
-        view.addSubview(child.view)
         child.didMove(toParent: self)
```
Comment (mohammedDehairy) [inline AdyenDropIn/Presentation/WrapperViewController.swift:56]: this has to be after `view.addSubview(child.view)`, like this:

```
addChild(child)
view.addSubview(child.view)
child.didMove(toParent: self)
setupChildLayout()
```

this is more accurate

File: AdyenDropIn/Presentation/WrapperViewController.swift (hunk 1 above)
Comment (nauaros) [reply AdyenDropIn/Presentation/WrapperViewController.swift]: Remove, thanks!

File: AdyenDropIn/Presentation/WrapperViewController.swift (hunk 2 above)
Comment (nauaros) [reply AdyenDropIn/Presentation/WrapperViewController.swift]: I agree, done

**Review by mohammedDehairy — APPROVED** [review]

**Review by descorp — APPROVED** [review]

**Pull request merged**